### PR TITLE
Fix captcha

### LINKED ISSUE #16: Captcha not working on development
I was having issues with the captcha not validating. I was already in the process of fixing this by making the sitekey and secret configurable.

### PR DESCRIPTION
**Pull request by ferrydeboer**
Made captcha keys configurable and added Google captcha v2 default keys.

### CODE REVIEW

**Comment by ferrydeboer** [comment]
Haha, it's completely against my principles to merge my own PR's. But for the sake of getting this master stable asap I guess I make an exception.

File: .env.example [hunk 1]
```diff
@@ -1 +1,3 @@
 MYSQLCONNSTR_localdb=db=test;HOST=mariadb;user=test;password=test
+captchaSecret="6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
```
Comment (xvilo) [inline .env.example:2]: Please make them all upper case next time

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 1
-          ref: ${{ github.head_ref }}
+          ref: ${{ github.event.pull_request.head.sha }}
```
Comment (xvilo) [inline .github/workflows/lint.yml:10]: Why this change?

File: heroForm.php [hunk 3]
```diff
@@ -38,4 +38,4 @@
     
 
     
-    echo $twig->render('heroForm.html', []);
+    echo $twig->render('heroForm.html', ["captchaSiteKey"=>getenv("captchaSiteKey")]);
```
Comment (xvilo) [inline heroForm.php:41]: Please use single quotes and  add space between =>

**Review by xvilo — COMMENTED** [review]
Same here

**Comment by xvilo** [comment]
Sorry for bitching on an already merged request

File: .github/workflows/lint.yml (hunk 2 above)
Comment (ferrydeboer) [reply .github/workflows/lint.yml:10]: Because it was failing to checkout the PR's. It worked on my own repo. I couldn't really explain it since I'm pretty sure I've seen it working you rinitial PR. I found this line in the readme of the action, tried it and it worked. this I stuck with it. If you have a better solution I'm open for it.
